docs/extensions.md

```diff
@@ -2,18 +2,113 @@
 
 ## Creating extensions
 
-### Extension template
+OCDS extensions follow the structure of the [standard extension template](https://github.com/open-contracting/standard_extension_template). The template copies the structure of the core schema (except for _versioned-release-validation-schema.json_, see below) and allows you to add or update fields in those places where the extension should modify the schema using [JSON merge patch](https://tools.ietf.org/html/rfc7396).
 
-[Extension template](https://github.com/open-contracting/standard_extension_template)
+The core repository for OCDS has 4 [schema files](https://github.com/open-contracting/standard/tree/master/standard/schema):
+
+* release-schema.json
+* release-package-schema.json
+* record-package-schema.json
+* versioned-release-validation-schema.json
+
+_versioned-release-validation-schema.json_ is produced programmaticallly (see [Make validation schema](../deployment/standard-live#make-validation-schema) in the standard deployment section), which means that you don't have to author it manually when creating an extension.
+
+**TODO**: Mechanisn for extension authors to create _versioned-release-validation-schema.json_ in their extensions.
```

Comment:
In #19, after discussion in https://github.com/open-contracting/standard/issues/517, we agreed to just delete `versioned-release-validation-schema.json` from all extensions. So, I think we can remove the discussion of that file from here.